File: src/androidTest/java/pt/joaomneto/titancompanion/TestFF.java
package pt.joaomneto.titancompanion;


import android.support.test.filters.LargeTest;
import android.support.test.runner.AndroidJUnit4;

import org.junit.runner.RunWith;

import pt.joaomneto.titancompanion.consts.FightingFantasyGamebook;

import static pt.joaomneto.titancompanion.consts.FightingFantasyGamebook.FREEWAY_FIGHTER;

@LargeTest
@RunWith(AndroidJUnit4.class)
public class TestFF extends TestST {

    @Override
    protected FightingFantasyGamebook getGamebook() {
        return FREEWAY_FIGHTER;
    }


}

File: src/androidTest/java/pt/joaomneto/titancompanion/phase1/TestFF.kt
package pt.joaomneto.titancompanion.phase1

import androidx.test.ext.junit.runners.AndroidJUnit4
import androidx.test.filters.LargeTest
import org.junit.runner.RunWith
import pt.joaomneto.titancompanion.consts.FightingFantasyGamebook.FREEWAY_FIGHTER

@LargeTest
@RunWith(AndroidJUnit4::class)
class TestFF : TestST() {

    override val gamebook = FREEWAY_FIGHTER
}
